site/ja/docs/privacy-sandbox/status/index.md

```diff
@@ -1,99 +1,168 @@
+
+- [オリジントライアルとスケジュール](https://blog.chromium.org/2021/09/user-agent-reduction-origin-trial-and-dates.html)
+
+- [オリジントライアルに登録](/origintrials/#/view_trial/-7123568710593282047)
+
+- [デモ](https://uar-ot.glitch.me/)
+
+- [UA-CH の提案](https://github.com/WICG/ua-client-hints)
+
+- [UA 文字列から UA-CH に移行](https://web.dev/migrate-to-ua-ch/)するためのチュートリアル
+
+- 詳細: [User-Agent の削減](/docs/privacy-sandbox/user-agent/)
+
+## [FLoC](/docs/privacy-sandbox/status/#floc)
 
-- 最初の[オリジントライアル](https://web.dev/origin-trials)は現在終了しています。更新については、[Intent to Experiment](https://groups.google.com/a/chromium.org/g/blink-dev/c/MmijXrmwrJs)を参照してください。
-- [初期バージョンのデモ](https://floc.glitch.me/)（オリジントライアルは終了しました）。
-- [Blinkでの状況](https://groups.google.com/a/chromium.org/g/blink-dev/search?q=floc)。
-- [APIの提案](https://github.com/WICG/floc)は、 [WICG](https://www.w3.org/community/wicg/)および利害関係者グループと議論中です。
-- [GitHub](https://github.com/WICG/floc) ：APIの質問とディスカッションについては[問題](https://github.com/WICG/floc/issues)を参照してください。
-- [Chromeプラットフォームでの状況](https://www.chromestatus.com/features/5710139774468096)。
-- 詳細： [FLoCとは？](https://web.dev/floc/)
+Topics API に置き換えられました。
 
-## FLEDGE
+- 最初の[オリジントライアル](https://web.dev/origin-trials)は終了しました。最新情報については、[Intent to Experiment](https://groups.google.com/a/chromium.org/g/blink-dev/c/MmijXrmwrJs) をご覧ください。
 
-[TURTLEDOVEの](https://github.com/WICG/turtledove)から生まれたAPI。
+- [初期バージョンのデモ](https://floc.glitch.me/)（オリジン トライアルは終了しました）。
+
+- [Blink のステータス](https://groups.google.com/a/chromium.org/g/blink-dev/search?q=floc)。
+
+- [API の提案](https://github.com/WICG/floc)は、[WICG](https://www.w3.org/community/wicg/)、インタレスト グループと検討中です。
+
+- [GitHub](https://github.com/WICG/floc): API に関する質問とディスカッションについては、[イシュー](https://github.com/WICG/floc/issues)をご覧ください。
+
+- [Chrome プラットフォームのステータス](https://www.chromestatus.com/features/5710139774468096)。
+
+- 詳細: [FLoC とは](https://web.dev/floc/)
+
+## [FLEDGE](/docs/privacy-sandbox/status/#fledge)
+
+[TURTLEDOVE](https://github.com/WICG/turtledove) の子孫。
+
+- [Intent to Experiment](https://groups.google.com/a/chromium.org/g/blink-dev/c/0VmMSsDWsFg)。
 
 - [Intent to Prototype](https://groups.google.com/a/chromium.org/g/blink-dev/c/w9hm8eQCmNI/m/LqT59250CAAJ)。
-- [Blinkで状況](https://groups.google.com/a/chromium.org/g/blink-dev/search?q=fledge)。
-- [APIの提案](https://github.com/WICG/turtledove/blob/main/FLEDGE.md)は、 [WICG](https://www.w3.org/community/wicg/)および利害関係者グループと議論中です。
-- [GitHub](https://github.com/WICG/turtledove/blob/main/FLEDGE.md) ：APIの質問とディスカッションについて[は、TURTLEDOVEの問題](https://github.com/WICG/turtledove/issues)を参照してください。
 
-<br>
+- [Blink のステータス](https://groups.google.com/a/chromium.org/g/blink-dev/search?q=fledge)。
 
----
+- [API の提案](https://github.com/WICG/turtledove/blob/main/FLEDGE.md)は、[WICG](https://www.w3.org/community/wicg/)、インタレスト グループと検討中です。
+
+- [GitHub](https://github.com/WICG/turtledove/blob/main/FLEDGE.md) : API に関する質問とディスカッションについては、[TURTLEDOVE のイシュー](https://github.com/WICG/turtledove/issues)をご覧ください。
+- ```
```

Comment:
Please remove this empty code block.

Reply:
@agektmr I've removed the empty code block